pkg/virt-controller/watch/migration.go

```diff
@@ -865,7 +865,7 @@ func (c *MigrationController) handlePreHandoffMigrationCancel(migration *virtv1.
 		return fmt.Errorf("cannot delete pending target pod %s/%s for migration although migration is aborted", pod.Name, pod.Namespace)
 	}
 
-	reason := "migration canceled"
+	reason := "migration canceled and pod %s is deleted"
 	log.Log.Object(vmi).Infof("Deleted pending migration target pod %s/%s with uuid %s for migration %s with uuid %s with reason [%s]", pod.Namespace, pod.Name, string(pod.UID), migration.Name, string(migration.UID), reason)
```

Comment:
I think here you are still printing `reason` without the string

Reply:
> I think here you are still printing `reason` without the string

Yes, here is a problem. I will update.